client/src/components/Dashboard/TaskItem/TaskItem.jsx

```diff
@@ -63,38 +67,56 @@ class TaskItem extends Component {
 
     return (
       <Container
-        className="task-item-container"
+        className="p-0"
         fluid="true"
       >
         <Container
           id={id}
           fluid="false"
+          data-test="taskContainer"
           draggable="true"
-          data-test="taskName"
           className={getTaskStyleByPriority(priority)}
           onClick={this.openTaskDetails}
           onDragStart={e => dragStart(e)}
           onDragEnd={e => dragEnd(e)}
         >
-          <span className={getTaskStyleByStatus(status)}>
-            {taskName}
-          </span>
-          <Button
-            variant="light"
-            className="delete-button"
-            size="sm"
-            onClick={this.deleteTaskHandle}
-            onMouseLeave={() => this.setState({ isDeleted: false })}
+          <Col xs={11}>
+            <Row>
+              <Container
+                className={getTaskStyleByStatus(status)}
+                data-test="taskName"
+              >
+                <span>
+                  {taskName}
+                </span>
+              </Container>
+            </Row>
+            <Row>
+              <Container>
+                <Badge variant="secondary">0 / 0</Badge>
+              </Container>
+            </Row>
+          </Col>
+          <Col
+            xs={1}
+            className="d-flex align-items-center m-0 p-0"
           >
-            {!isDeleted
-              ? <img src={del} alt={del} className="delete-icon" draggable="false" />
-              : <img src={accept} alt={accept} className="accept-icon" draggable="false" />}
-          </Button>
+            <Button
+              variant="light"
+              className="delete-button"
+              size="sm"
+              onClick={this.deleteTaskHandle}
+              onMouseLeave={() => this.setState({ isDeleted: false })}
+            >
+              {!isDeleted
+                ? <img src={del} alt={del} className="delete-icon" draggable="false" />
```

Comment:
I know we already discussed this... but still, **alt** attribute specifies an alternate text for an image, if the image cannot be displayed. In your case, at the moment it will display URL to image, instead of text saying what this button is for.

> The **alt** attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the src attribute, or if the user uses a screen reader).
https://www.w3schools.com/tags/att_img_alt.asp

I'd recommend you to use it as it was intended.